appveyor.yml

```diff
@@ -95,22 +95,9 @@ test_script:
   - coverage erase
   - del /Q coverage.xml
 
-  - ps: |
-      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache
-      $testsExitCode = $lastexitcode
-      & coverage report
-      & coverage xml
-
-      $wc = New-Object 'System.Net.WebClient'
-      $wc.UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\integrationtests.xml))
-
-      & codecov --no-color -X gcov -F integrationtests -e PYTHON_INSTALL
-
-      if ($testsExitCode -ne 0) {exit $testsExitCode}
-
```

Comment:
This might look odd, but with `pytest` convention, we can run all of the tests at once with a single command because `pytest` already knows what the test files are called.